src/error.rs

```diff
@@ -0,0 +1,11 @@
+use std::result;
```

Comment:
Ich bin ein Fan davon, die Fehler da zu definieren, wo sie gebraucht werden. Was ist der Grund dafür, sie in eine separate Datei zu verlegen? :)

Reply:
mh das ist eigentlich best-practice in Rust das in eine eigene Datei zu legen. Ich brauche halt ein error enum um die verschiedenen Fehler von der Datenbank, HTTP server, uuid etc zu composen und als Antwort in Rocket zurückgeben zu können.

Reply:
Ich hab die mal verschoben. Was hältst du davon?

Reply:
Ich bin mir gerade auch garnicht mehr sicher ob wir überhaupt ein Error enum brauchen, da der Responder trait für alle variants implementiert sein muss, e.g. wir können fast nur Strings composen. Dann können wir das mit dem enum auch direkt sein lassen und in den `map_err` immer ein String erzeugen.